.github/workflows/pr-check.yaml

```diff
@@ -118,6 +123,11 @@ jobs:
           run_install: false
           package_json_file: ./podman-desktop/package.json
 
+      - name: add macadam package authentication
```

Comment:
It's not clear to me why we need to add macadam auth here (as well as in the extension below)?

Reply:
Ah you're right. Only need to add it to the extension below. I'll remove this one. 

I got confused since I saw "pnpm install" below it and thought it was the extension.